aea/helpers/dialogue/base.py

```diff
@@ -102,6 +121,16 @@ def is_self_initiated(self) -> bool:
         """Check whether the agent initiated the dialogue."""
         return self._is_self_initiated
 
+    @property
+    def last_incoming_message(self) -> Optional[Message]:
+        """Get the last incoming message."""
+        return self._incoming_messages[-1]
```

Comment:
what if this is called when `self._incoming_messages` is an empty list? The property should check this corner case.